src/backend/TrafficCourts/Citizen.Service/Models/Dispute/NoticeOfDispute.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Swashbuckle.AspNetCore.Annotations;
+using System.Text.Json.Serialization;
+using TrafficCourts.Common.Converters;
+
+namespace TrafficCourts.Citizen.Service.Models.Dispute
+{
+    public class NoticeOfDispute
+    {
+        public string? TicketNumber { get; set; }
+        public string? ProvincialCourtHearingLocation { get; set; }
+        public DateTime? IssuedDate { get; set; }
+        public string? Surname { get; set; }
+        public string? GivenNames { get; set; }
+        public string? Address { get; set; }
+        public string? City { get; set; }
+        public string? Province { get; set; }
+        public string? PostalCode { get; set; }
+        public string? HomePhoneNumber { get; set; }
+        public string? WorkPhoneNumber { get; set; }
+        public string? EmailAddress { get; set; }
+        public IList<DisputedCount> DisputedCounts { get; set; } = new List<DisputedCount>();
+        public bool RepresentedByLawyer { get; set; }
+        public LegalRepresentation? legalRepresentation { get; set; }
+        public string? InterpreterLanguage { get; set; }
+        public int NumberOfWitness { get; set; }
+        public string? FineReductionReason { get; set; }
+        public string? TimeToPayReason { get; set; }
+        public bool CitizenDetectedOcrIssues { get; set; }
+        public ViolationTicket ViolationTicket { get; set; } = new();
+        public string? OcrKey { get; set; }
+    }
+
+    public class DisputedCount
+    {
+        public Plea Plea { get; set; }
+        public int Count { get; set; }
+        public bool RequestTimeToPay { get; set; }
+        public bool RequestReduction { get; set; }
+        public bool AppearInCourt { get; set; }
```

Comment:
If the person is pleading not guilty this technically should not be set.  If pleading not guilty, they have to appear in court.